scripts/publish.sh

```diff
@@ -37,20 +58,51 @@ function processCommandLineArgs {
 
 function release {
   BUMP=$(npx -p conventional-changelog-angular -p conventional-recommended-bump -c 'conventional-recommended-bump -p angular')
-  VERSION=$(yarn version --no-git-tag-version --new-version ${1:-$BUMP})
+  # jq to get version from yarn https://jqplay.org/s/LqIS_qy2MBx
+  VERSION=$(yarn version --no-git-tag-version --new-version ${release_type:-$BUMP} --json | jq -r 'select(.data | startswith("New version")).data | split(": ")[1]')
 
   npx conventional-changelog-cli -p angular -i CHANGELOG.md -s
-  git add CHANGELOG.md
-  git commit -m "docs(CHANGELOG): $VERSION"
-  yarn version --force --allow-same-version --new-version $VERSION --message "chore(release): %s"
-  git push --follow-tags
-  npx conventional-github-releaser -p angular
 
-  echo "Beginning release $PKG_NAME@$VERSION"
-  yarn publish --access public --non-interactive
+  if "$dry_run"; then
+    echo
+    echo "In non-dry-run, will run the following commands"
+    echo "git add CHANGELOG.md"
+    echo git commit -m "docs(CHANGELOG): $VERSION"
```

Comment:
shouldn't this be be in quotes `git commit ...`?

Reply:
It doesn't *need* it, but I think worth adding for consistency.